week-1-ts-ecosystem/w01e01-typescript-basics/package.json

```diff
@@ -1,5 +1,16 @@
 {
   "name": "w01e01-typescript-basics",
   "version": "1.0.0",
-  "scripts": {}
+  "main": "main.js",
+  "scripts": {
+    "start": "parcel index.html watch"
```

Comment:
Fajnie że dodałaś `parcel` @monikadembek - jednak pamiętaj że z pkt. widzenia zadania nie było to koniczne :)
bardziej zależy mi na Twoim zrozumieniu dodania tutaj `tsc` (typescript)

Żeby jasne było to że nie potrzeba `parcel` żeby mieć `typescript` w projekcie 👍